scripts/modules/b2b-account/quotes.js

```diff
@@ -21,12 +21,14 @@ define([
     CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
     ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
     PagingViews, EditableView, QuoteModels) {
-
+        var NAMESTRING = "name cont ";
+        var DateString = "expirationdate ge ";
```

Comment:
rename 'DateString' to maybe something like 'expirationDateFilter'

Reply:
done